File: Tameenk/src/app/home/models/insured.model.ts
export class Insured {
  constructor() {
      this.nationalId = '';
      this.birthDateMonth = null;
      this.birthDateYear = null;
      this.edcuationId = null;
      this.childrenBelow16Years = null;
      this.validationErrors = null
  }
  /**
   * @property {string} nationalId - The Insured National identifier.
   */
  nationalId: string;
  /**
   * @property {number} birthDateMonth - The Insured birth Date Month id.
   */
  birthDateMonth: number;
  /**
   * @property {number} birthDateYear - The Insured birth Date Year id.
   */
  birthDateYear: number;
  /**
   * @property {number} edcuationId - The Insured edcuation level id.
   */
  edcuationId: number;
  /**
   * @property {number} childrenBelow16Years - The Insured childrens Below 16 Years Count.
   */
  childrenBelow16Years: number;
  validationErrors: string[];
}

File: Services_Core/Inquiry/Tameenk.Services.InquiryGatewayCore/Models/InsuredModel.cs
using Newtonsoft.Json;
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Web;
using Tameenk.Resources.Inquiry;

namespace Tameenk.Services.InquiryGateway.Models
{
    /// <summary>
    /// Represent the insured personal information required for inquiry.
    /// </summary>
    [JsonObject("insured")]
    public class InsuredModel
    {
        /// <summary>
        /// National ID / Iqama ID
        /// </summary>
        [JsonProperty("nationalId")]
        [Required(ErrorMessageResourceType =typeof(SubmitInquiryResource), ErrorMessageResourceName = "NationalIdRequired")]
        [Range(1000000000, 99999999999)]
        public string NationalId { get; set; }

        /// <summary>
        /// Customer birthdate month.
        /// </summary>
        [JsonProperty("birthDateMonth")]
        [Required]
        [Range(1, 12)]
        public byte BirthDateMonth { get; set; }


        /// <summary>
        /// Customer birth date year.
        /// </summary>
        [JsonProperty("birthDateYear")]
        [Required]
        public short BirthDateYear { get; set; }
        
        /// <summary>
        /// Insured Education identifier.
        /// </summary>
        [Required]
        [JsonProperty("edcuationId")]
        public int EducationId { get; set; }

        /// <summary>
        /// Number of children under age 16 years.
        /// </summary>
        [JsonProperty("childrenBelow16Years")]
        public int ChildrenBelow16Years { get; set; }
        
    }
}